promise-it-wont-hurt-solutions/src/05-always-asynchronous.js

```diff
@@ -0,0 +1,8 @@
+const promise = new Promise((fulfill, reject) => {
+	fulfill('PROMISE VALUE')
+});
+
+promise
+	.then(console.log)
```

Comment:
why is there so much indentation here? Double check your text editor and linter.